udev rule to access HID devices on Linux & friends (#379)
NordicPlayground/fw-nrfconnect-nrf: scripts/hid_configurator/README.rst

	pip3 install --user -r requirements.txt

This installs:

* HIDAPI library - on Windows provided by `hidapi.dll` or `hidapi-x64.dll`
* pyhidapi Python wrapper - installed from GitHub repository with additional patches included. The version on PyPI is obsolete and will not work.

> On Linux, if you want to be able to call Python script without root rights,

Reviewer: ```suggestion
On Linux, to call the Python script without root rights,
```

install the provided udev rule `99-hid.rules` by copying it to

> `/etc/udev/rules.d` and re-plugging the device.

Reviewer: ```suggestion
`/etc/udev/rules.d` and replugging the device.
```
